app-misc/jigdo/jigdo-0.7.3.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Ease the distribution of files"
+DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to ease the distribution \
+of very large files over the internet, for example CD or DVD images. Its aim is \
+to make downloading the images as easy for users as a click on a direct \
+download link in a browser, while avoiding all the problems that server \
+administrators have with hosting such large files."
+HOMEPAGE="http://atterer.org/jigdo/"
+COPYRIGHT="2001-2006 Richard Atterer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://atterer.org/sites/atterer/files/2009-08/jigdo/jigdo-$portVersion.tar.bz2"
+CHECKSUM_SHA256="875c069abad67ce67d032a9479228acdb37c8162236c0e768369505f264827f0"
+PATCHES="jigdo-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	jigdo$secondaryArchSuffix = $portVersion
+	cmd:jigdo_file$secondaryArchSuffix = $portVersion
+	cmd:jigdo_lite$secondaryArchSuffix = $portVersion
+	cmd:jigdo_mirror$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
```

Comment:
`jigdo-lite` and `jigdo-mirror` both need wget, so we should add a `cmd:wget` to `REQUIRES`.

I don't know yet what's the recommended place, I mean before of after the `lib:lib*` lines.

I've just compared the output of `wget --help` and `wget-x86 --help` and they are identical (except for the `argv[0]`) so I'm quite sure we don't need the `$secondaryArchSuffix`.
BTW, if, for some reason, we wanted to use the wget_x86, we would probably have to patch `jigdo-lite` and `jigdo-mirror`.

Reply:
Maybe put it before "lib:..."? It would sort alphabetically then

Reply:
Why not. IDK. I'm trying to see if there is a trend on this question :)
BTW, maybe we could add a debuginfo package?
```
defineDebugInfoPackage jigdo$secondaryArchSuffix \
	$commandBinDir/jigdo-file
```

BTW, it would be really nice to use the commandBinDir "trick".